app/order-history/page.tsx

```diff
@@ -0,0 +1,381 @@
+"use client";
+
+import { useState, useMemo } from "react";
+import { Table } from "@/components/ui/table";
+import { Badge } from "@/components/ui/badge";
+import { Input } from "@/components/ui/input";
+import { X } from "lucide-react";
+
+
+
+const orderData = [
```

Comment:
lets do this as of now but this should have been type annotated and extracted to a different page. but its fine as of now